src/components/molecules/Slider/index.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import Button from '../../atoms/Button';
+import CarouselText from '../CarouselText';
+import './style.scss';
+
+const Slider = ({ title, subtitle, images }) => {
+  const slides = [
+    <img key="1" src={images && require(`../../../images/${images[2].src}`).default} alt={images && images[2].alt} className="slider__slide__img"/>,
+    <img key="2" src={images && require(`../../../images/${images[0].src}`).default} alt={images && images[0].alt} className="slider__slide__img"/>,
+    <img key="3" src={images && require(`../../../images/${images[1].src}`).default} alt={images && images[1].alt} className="slider__slide__img"/>,
+    <img key="4" src={images && require(`../../../images/${images[2].src}`).default} alt={images && images[2].alt} className="slider__slide__img"/>,
+    <img key="5" src={images && require(`../../../images/${images[0].src}`).default} alt={images && images[0].alt} className="slider__slide__img"/>,
+  ];
+  const [transform, setTransform] = useState(-100);
+  const [transition, setTransition] = useState('transitionOn');
+  const [step, setStep] = useState(1);
+
+  const dropDownClickLeft = () => {
+    setStep(step - 1);
+    if (step < 1) {
+      setStep(slides.length - 1);
+    }
+    setTransform(-100 * (step - 1));
+    if (step === 1 && transition === 'transitionOn') {
+      setTimeout(() => {
+        setTransition('transitionOff');
+        setStep(slides.length - 2);
+        setTransform(-100 * (slides.length - 2));
+      }, 300);
+    } else if (step === 1 && transition === 'transitionOff') {
+      setTransition('transitionOn');
+      setTimeout(() => {
+        setTransition('transitionOff');
+        setStep(slides.length - 2);
+        setTransform(-100 * (slides.length - 2));
+      }, 300);
+    } else {
+      setTransition('transitionOn');
+    }
+  };
+
+  const dropDownClickRight = () => {
+    setStep(step + 1);
+    if (step === slides.length - 1) {
+      setStep(1);
```

Comment:
use here as well variables and not setState, that will reduce amount of  rerenders